app/models/friendship.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Friendship < ApplicationRecord
```

Comment:
Since `Friendship` is taking care of both friend requests and actual friendships between users, we might need an instance method here that performs the logic for accepting a request. This is so later on the controller won't have to carry the burden that much:

```ruby
def accept
  # write the logic here
end
```

In the controller `update` action later on then:

```ruby
def update
  @friendship = Friendship.find(params[:id]) # supposing your params will carry this id
  
  if @friendship.accept
   # do something
  else
   # do something else
  end
end
```